Formula/mothur.rb

```diff
@@ -0,0 +1,25 @@
+class Mothur < Formula
+  desc "16s analysis software"
+  homepage "https://www.mothur.org/"
+  url "https://github.com/mothur/mothur/archive/v1.39.5.tar.gz"
+  sha256 "9f1cd691e9631a2ab7647b19eb59cd21ea643f29b22cde73d7f343372dfee342"
+  revision 2
+  head "https://github.com/mothur/mothur.git"
+  # tag "bioinformatics"
+  # doi "10.1128/AEM.01541-09"
+
+  depends_on "boost"
+
+  def install
+    boost = Formula["boost"]
+    inreplace "Makefile", '"\"Enter_your_boost_library_path_here\""', boost.opt_lib
```

Comment:
This will need to build without string replacements.

Reply:
Thanks for the review. Both this and the `make install` are upstream issues. Feel free to put this PR on hold for now